src/Cargo.toml

```diff
@@ -16,6 +16,14 @@ members = [
   "tools/rust-installer",
   "tools/cargo",
   "tools/rls",
+  "tools/rls/test_data/borrow_error",
+  "tools/rls/test_data/completion",
+  "tools/rls/test_data/find_all_refs",
+  "tools/rls/test_data/find_all_refs_no_cfg_test",
+  "tools/rls/test_data/goto_def",
+  "tools/rls/test_data/highlight",
+  "tools/rls/test_data/hover",
+  "tools/rls/test_data/rename",
```

Comment:
Hm [this logic](https://github.com/rust-lang/cargo/blob/82733b01471a2c62bb1cec966d888c52ff118914/src/cargo/core/workspace.rs#L564-L582) makes me think that `exclude` could work. Are you sure the above doesn't work? I've just added a test case to Cargo locally to and this works as expected.